src/Rule/Each.php

```diff
@@ -37,20 +37,27 @@ protected function validateValue($value, ValidationContext $context = null): Res
             return $result;
         }
 
-        foreach ($value as $item) {
+        foreach ($value as $index => $item) {
             $itemResult = $this->rules->validate($item, $context);
-            if ($itemResult->isValid() === false) {
-                foreach ($itemResult->getErrors() as $error) {
-                    $result->addError(
-                        $this->formatMessage(
-                            $this->message,
-                            [
-                                'error' => $error,
-                                'value' => $item,
-                            ]
-                        )
-                    );
+            if ($itemResult->isValid() === true) {
```

Comment:
```suggestion
            if ($itemResult->isValid()) {
```

Reply:
But it was strictly checked for `false` before, I just refactored it to continue early to reduce indentation level and improve readability (following best practices). A bit confused. But it seems to be safe here because of using `bool`.

Reply:
:heavy_check_mark: